new/LIT7257Lightning.xml

```diff
@@ -0,0 +1,75 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7257Lightning" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">... ያውርድ፡ መብረቀ፡ በጊዜ፡ ዝናም፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">yāwrǝd mabraqa ba-gize zǝnām</title>
```

Comment:
…

Reply:
I mean that if you mark versions (transcription/translation) as corresp to something they should ideally correspond